$OpenBSD$

OpenBSD does not need the upstream nullary-to-object override for this JSON
derivation.

--- ../simplexmq-stable/src/Simplex/RemoteControl/Types.hs
+++ ../simplexmq-stable/src/Simplex/RemoteControl/Types.hs
@@ -155,7 +155,7 @@
 data RCCtrlHello = RCCtrlHello {}
   deriving (Show)
 
-$(JQ.deriveJSON defaultJSON {J.nullaryToObject = True} ''RCCtrlHello)
+$(JQ.deriveJSON defaultJSON ''RCCtrlHello)
 
 -- | Long-term part of controller (desktop) connection to host (mobile)
 data RCHostPairing = RCHostPairing
